tests/test_sensor.py

```diff
@@ -80,6 +86,87 @@ def status_func(value):
     assert sensor.status == Sensor.Status.NOMINAL
 
 
+class MyEnum(enum.Enum):
+    ZERO = 0
+    ONE = 1
+
+
+class OtherEnum(enum.Enum):
+    ABC = 0
+    DEF = 1
+
+
+@pytest.mark.parametrize(
+    "sensor_type,compatible_value",
+    [
+        (bool, True),
+        (int, 1_2_3_4),
+        (float, Fraction(1234678)),
+        (float, Fraction(12345678)),
```

Comment:
Not sure why you have two of these. I'd suggest that
(a) you have a case where you use an actual float
(b) in the Fraction case, you use a non-integral Fraction (but use a power of two as the denominator, as otherwise the `==` test may fail due to rounding)

Reply:
Ah, I previously had this as two types of `numpy.floatXX`; naively did a search-and-replace. Either way, noted all around.